src/main/groovy/org/shipkit/internal/gradle/AutoDiscoverGradlePluginsPlugin.java

```diff
@@ -0,0 +1,61 @@
+package org.shipkit.internal.gradle;
+
+import com.gradle.publish.PluginBundleExtension;
+import com.gradle.publish.PluginConfig;
+import org.gradle.api.Action;
+import org.gradle.api.Plugin;
+import org.gradle.api.Project;
+import org.gradle.api.file.FileTree;
+import org.gradle.api.plugins.JavaPluginConvention;
+import org.gradle.api.tasks.util.PatternSet;
+
+import java.io.File;
+import java.util.Set;
+
+/**
+ * This plugin discovers gradle plugins and adds them to the {@link PluginBundleExtension}.
```

Comment:
Can you write a few words why thy plugin is useful? E.g. that maintaining plugins manually is PITA, etc. Let's make the docs great!